frontend-react/src/components/Content/MarkdownRenderer.tsx

```diff
@@ -26,6 +28,29 @@ type MarkdownContentProps = {
     markdownUrl: string;
 };
 
+type ReactMarkdownComponentsProp = Exclude<Options["components"], undefined>;
```

Comment:
Normally I'm a little wary of maintaining our own types for an external library, but this makes sense since we may want to extend other components, and it'd be a hassle to extend each other individually.  But would you mind adding a comment for why this is needed and linking to the [extra props section](https://github.com/remarkjs/react-markdown#appendix-b-components) for more context?